packages/block-editor/src/components/inspector-controls-tabs/index.js

```diff
@@ -6,17 +6,23 @@ import { TabPanel } from '@wordpress/components';
 /**
  * Internal dependencies
  */
-import { TAB_SETTINGS, TAB_APPEARANCE } from './utils';
+import { TAB_SETTINGS, TAB_APPEARANCE, TAB_LIST_VIEW } from './utils';
```

Comment:
Can't this tab be an ... extensibility tab. Maybe the navigation block renders a list view, maybe another block renders something else?

Reply:
Yeah, it was discussed on https://github.com/WordPress/gutenberg/pull/45005 that it could work this way, but it's a little bit more complicated to implement and right now it's only for one block, so this starts with the easy option.